components/brave_wallet_ui/common/trezor/trezor-messages.ts

```diff
@@ -31,21 +31,35 @@ export type UnlockResponse = CommandMessage & {
   result: Boolean,
   error?: Unsuccessful
 }
+
+export type SignTransactionCommandPayload = CommonParams & EthereumSignTransaction
+
+export type SignTransactionCommand = CommandMessage & {
+  command: TrezorCommand.SignTransaction
+  payload: SignTransactionCommandPayload
+}
+
+export type SignTransactionResponse = Unsuccessful | Success<EthereumSignedTx>
+
 export type TrezorAccount = {
   publicKey: string
   serializedPath: string,
   fingerprint: number
 }
 export type TrezorError = {
   error: string,
-  code: string
+  code?: string | number
```

Comment:
Can we document here when the `code` is a string and when it's a number?

Reply:
I will remove this type in my next PR where I refactored keyrings https://github.com/brave/brave-core/pull/10924/files, that PR should be merged after this one